Core/src/main/java/com/paypal/android/core/graphql/fundingEligibility/models/FundingEligibilityResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.paypal.android.core.graphql.fundingEligibility.models
+
+import org.json.JSONObject
+
+data class FundingEligibilityResponse(
```

Comment:
this could be internal
```suggestion
internal data class FundingEligibilityResponse(
```